perf_dashboard/assets/js/alert_common_func.js

```diff
@@ -26,7 +26,7 @@ window.generateChart = function(chartID, modesData) {
       }
     },
     axisY: {
-      title: "p90 Latency Pattern in milliseconds",
+      title: "Latency Pattern (milliseconds)",
```

Comment:
"latency" is not meaningful without the percentile

Reply:
the X-axis title with P90 or P99 is in specified in html file, here is just common js file which stands for the Y-axis title.